sympy/combinatorics/pc_groups.py

```diff
@@ -8,16 +8,17 @@ class PolycyclicGroup(DefaultPrinting):
     is_group = True
     is_solvable = True
 
-    def __init__(self, pc_sequence, pc_series):
-        self.pc_series = pc_series
+    def __init__(self, pc_sequence):
+        #self.pc_series = pc_series
         self.pcgs = pc_sequence
 
     def relative_order(self):
-        rel_orders = []
-        for i in range(len(self.pc_series)-1):
-            G = self.pc_series[i]
-            H = self.pc_series[i+1]
-            rel_orders.append(G.order()//H.order())
+        # rel_orders = []
+        # for i in range(len(self.pc_series)-1):
+        #     G = self.pc_series[i]
+        #     H = self.pc_series[i+1]
+        #     rel_orders.append(G.order()//H.order())
+        rel_orders = [g.order() for g in self.pcgs]
```

Comment:
These are not *relative* orders.

Reply:
Yes, but I think for the current tests that we've added they will be equal to the relative order!